F/FMM3D/build_tarballs.jl

```diff
@@ -30,24 +30,26 @@ cd FMM3D/
 
 touch make.inc
 
-# clang has some issue with openmp (lomp not found)
-# force gcc
-if [[ ${target} = *apple* || ${target} = *freebsd* ]]; then
-  export CC="gcc"
-fi
 
 echo "CC=${CC}" >> make.inc;
 echo "CXX=${CXX}" >> make.inc;
 echo "FC=${FC}" >> make.inc;
 
 if [[ ${target} = *mingw* ]]; then
-    echo "FFLAGS= -fPIC -O3 -fno-asynchronous-unwind-tables -funroll-loops" >> make.inc;
+    echo "FFLAGS= -fPIC -O3 -fno-asynchronous-unwind-tables -funroll-loops -std=legacy" >> make.inc;
 else
-    echo "FFLAGS= -fPIC -O3 -funroll-loops" >> make.inc;
+    echo "FFLAGS= -fPIC -O3 -funroll-loops -std=legacy" >> make.inc;
 fi
 
 export OMPFLAGS="-fopenmp"
 export OMPLIBS="-lgomp"
```

Comment:
I'm not entirely sure `-lgomp` or `-lomp` are necessary when already using `-fopenmp`.